types/postcss-less/postcss-less-tests.ts

```diff
@@ -40,31 +51,40 @@ less.parse(lessCode).walkAtRules(atRule => {
     }
 });
 
-less.parse(lessCode).walkRules(rule => {
+less.parse('.a:extend(.b) {}').walkRules(rule => {
```

Comment:
[note] Changed to the valid test data. We can check it at runtime as follows:

`cd types/postcss-less && npm i postcss-less && npx ts-node postcss-less-tests.ts`